haystack/components/routers/zero_shot_text_router.py

```diff
@@ -191,7 +191,6 @@ def from_dict(cls, data: Dict[str, Any]) -> "TransformersZeroShotTextRouter":
             deserialize_hf_model_kwargs(data["init_parameters"]["huggingface_pipeline_kwargs"])
         return default_from_dict(cls, data)
 
-    @component.output_types(documents=Dict[str, str])
     def run(self, text: str):
```

Comment:
I am confused either both of the run methods should specify return type or neither right?

```suggestion
    def run(self, text: str) -> Dict[str, str]:
```

Reply:
ahh fair point, sorry about that